app/lib/actions/pulp/abstract_async_task.rb

```diff
@@ -79,7 +79,7 @@ def external_task
         output[:pulp_tasks]
       end
 
-      def cancel!
+      def cancel!(raise_error = true)
```

Comment:
Does `raise_error` here mean that `cancel!` should not raise an error or that the code using this function intends to raise an error?